usr/lib/ica_s390_stdll/tok_struct.h

```diff
@@ -109,7 +109,7 @@ token_spec_t token_specific = {
     &token_specific_generic_secret_key_gen,
     // AES
     &token_specific_aes_key_gen,
-    NULL,                       // aes_xts_key_gen
+    &token_specific_aes_key_gen,
```

Comment:
Is this intended: lines 111 and 112 contain the same function ptr?

Reply:
Oh, no, not intended :-) Although it technically does not make a difference, since it just generates a random key in the length as specified by the caller. Will fix anyway .